keps/sig-windows/2578-windows-conformance/README.md

```diff
@@ -0,0 +1,418 @@
+
+### Sub-specifications of Windows Conformance (Kubernetes 1.23)
+
+The following subsets of verifications for Windows clusters further expand the way we define supportability for Windows clusters, but they are not considered part of the "core" Conformance specification.  These features may be *heavily* reliant on functionality which resides well outside of Kubernete's preview, for example: CNI Implementation, Container Runtime implementation, or Windows Server Edition + Configuration.
+
+#### Windows HostProcess Conformance
+
+Note that, HostProcess Containers have not yet been verified to support all of these features.
+Nevertheless, we define this specification because it is an obvious starting point for the long-term, meaningfull support of a full-fledged HostProcess feature in Kubernetes.
+
+- Ability to access the APIServer using pod mounted service accounts from a hostProcess pod.
+- Ability to create and manage host level networking (hcn) rules from a Windows hostProcess pod.
+- Ability to launch hostProcess containers which share the IP address of a Windows node.
+- Ability to launch hostProcess containers which can run other privileged Windows system API (to be specified further in the future).
+- Ability to utilize NT Authority/Network Service should be able to interact with host network objects
+
+#### Active Directory
+
+- Ability to run a pod as a GSMA User with only be able to access allowed resources
+- Ability to read and write from local storage using ActiveDirectory credentials
+- Observed lack of Ability to read and write from local storage when ActiveDirectory protected resources credentials are not present
+
+#### Network Policies
+
+Note that we do not include UDP, ipv6, in the initial definitions for NetworkPolicy conformance on Windows.  Since NetworkPolicy testing support on
+Windows clusters was just recently added to Kuberentes Core, we err on the side of a conservative definition for Windows NetworkPolicy which
+is sufficient to address the needs of *most* legacy network security applications.
+
+- Ability to protect IPv4 pods from acceessing other pods when TCP NetworkPolicys are present that block specific pod connectivity.
+- Ability to protect IPv4 pods from acceessing other pods when TCP NetworkPolicys are present that block specific namespace connectibity.
+
+#### Windows Service Proxying
+
+- Ability to support IPv6 and IPv4 services for accessing internal Linux pods from Windows pods over ClusterIP Endpoints.
+- Ability to support IPv6 and IPv4 services for accessing internal Linux pods from Windows pods over NodePort Endpoints.
+- Ability to support IPv6 and IPv4 services for accessing internal Windows pods from Linux pods over ClusterIP Endpoints.
+- Ability to support IPv6 and IPv4 services for accessing internal Windows pods from Linux pods over NodePort Endpoints.
```

Comment:
Is this only on some of the Windows versions?

Reply:
IPv6 dual stack is only supported on 2004+

Reply:
clarified